### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Link doesn't load for me.

Was this link tested before being added onto the lists ?

File: streams/us.m3u [hunk 1]
```diff
@@ -284,8 +284,10 @@ https://reflect-stream6-capsmedia.cablecast.tv/live/live.m3u8
 https://reflect-stream15-capsmedia.cablecast.tv/live/live.m3u8
 #EXTINF:-1 tvg-id="CaptitalOTBBetting.us" status="online",Captital OTB Betting (720p) [Not 24/7]
 https://d2up1hmow19bcd.cloudfront.net/livecf/liveracing/playlist.m3u8
-#EXTINF:-1 tvg-id="CartoonNetworkAsia.sg" status="timeout",Cartoon Network Asia (720p)
+#EXTINF:-1 tvg-id="CartoonNetworkAsia.sg" status="error",Cartoon Network Asia (720p)
 http://198.16.106.62:8278/streams/d/Cn/playlist.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkAsia.sg" status="online",Cartoon Network Asia
+http://otv.ftpbd.net/streams/123/index.m3u8
```
Comment (freearhey) [inline streams/us.m3u:290]: image

**Comment by bbbureneecartoonz** [comment]
REOPEN

**Comment by LaneSh4d0w** [comment]
I don't see why would I reopen this PR if the link doesn't work.

**Comment by bbbureneecartoonz** [comment]
> I don't see why would I reopen this PR if the link doesn't work.

REOPEN PLS

**Comment by LaneSh4d0w** [comment]
Then you'll have to tell me why should I reopen this PR ? You've found a new link ?

**Comment by bbbureneecartoonz** [comment]
> Then you'll have to tell me why should I reopen this PR ? You've found a new link ?

yess

**Comment by LaneSh4d0w** [comment]
Then please change it, and notify me once you've did the necessary changes.

**Comment by freearhey** [comment]
Author blocked for spamming: https://github.com/iptv-org/awesome-iptv/pull/144

**Pull request closed without merging**